modules/gapi/src/backends/streaming/gstreamingbackend.cpp

```diff
@@ -408,6 +434,15 @@ void GOCVUV::Actor::extractRMat(const cv::MediaFrame& frame, cv::RMat& rmat)
             });
             break;
         }
+        case cv::MediaFormat::GRAY:
+        {
+            rmat = cv::make_rmat<cv::gimpl::RMatMediaFrameAdapter>(frame,
+                [](const cv::GFrameDesc& d) { return cv::GMatDesc(CV_8U, 1, d.size); },
+                [](const cv::GFrameDesc& d, const cv::MediaFrame::View& v) {
+                    return cv::Mat(d.size, CV_8UC1, v.ptr[1], v.stride[1]);
```

Comment:
Do we need in division on half `d.size **/ 2**`  here?

Reply:
Don't see any reason. It's not UV plane.

Reply:
Could you clarify please how it is stored in array or put some references?

Reply:
Done